weather-app/requests/weather.js

```diff
@@ -25,8 +25,18 @@ const buildCoordinatesPair = (coordinates) => {
 
 const getWeather = (uri, searchInfo) => {
     request(uri, (error, response, body) => {
+        if (error !== null) {
+            console.log('A General Error occured when attempting to retrieve the weather forecast')
+            return
+        } 
+
         const bodyJson = JSON.parse(body)
-        if (bodyJson.error !== 'undefined') {
+    
+        if (bodyJson.error !== undefined) {
```

Comment:
can this be simplified to
`if (bodyJson.error)`???

Reply:
Good catch. It's done!